src/ui-test/lsp_extension_test.ts

```diff
@@ -76,10 +76,17 @@ describe('Language Support for Apache Camel extension', function () {
 		});
 
 		it('Language Support for Apache Camel started', async function () {
-			await driver.wait(until.elementLocated(By.id('redhat.vscode-apache-camel')), 35000);
+			const lsp = await driver.wait(until.elementLocated(By.id('redhat.vscode-apache-camel')), 35000);
 			await driver.wait(async () => {
-				const lsp = await new StatusBarExt().getLSPSupport().catch(() => '');
-				return lsp === LSP_STATUS_BAR_MESSAGE;
+				const text = await lsp.getText().catch(() => '');
+				try {
+					const codicon = await lsp.findElement(By.className('codicon'))
+					const klass = await codicon.getAttribute('class');
+					return text.startsWith('Camel LS') && klass.includes('codicon-thumbsup');
+				}
+				catch {
+					return false;
+				}
```

Comment:
created https://issues.redhat.com/browse/FUSETOOLS2-1252 to handle this in Test framework.

thanks @mlorinc fo rproviding the code to handle the status bar with an icon.